src/Security/AuthSamples.sln

```diff
@@ -0,0 +1,124 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
There needs to be a lot more project references to be added here to fix all the NU1105 errors.

Reply:
I thought we didn't need this if we have packages? I don't see any failures when I build the solution right now.